tools/broken-link-checker/pr.js

```diff
@@ -154,3 +179,57 @@ function dataFileToUrl(filename) {
 
   return `/${lookup[edition]}/${version}/`;
 }
+
+async function srcToUrls(src) {
+  let urls = [];
+
+  let files = await fg(`../../app/_data/docs_nav_*.yml`);
+  const navEntries = files
+    .map((f) => {
+      return yaml.load(fs.readFileSync(f, "utf8"));
+    })
+    .filter((n) => {
+      return n.generate;
+    });
```

Comment:
Should we cache this in a constant?
Right now it is loading, reading, and filtering all the files every time one of the modified files [starts](https://github.com/Kong/docs.konghq.com/blob/dd57c609ad54ebc978c1dc3bf20beb353288c00b/tools/broken-link-checker/pr.js#L56-L58) with `src`